Unity/Assets/Scripts/CharacterSelectionScript/LoadCharacter.cs

```diff
@@ -10,6 +10,8 @@
 /// </summary>
```

Comment:
![image](https://github.com/user-attachments/assets/07cfd256-985c-4503-934d-03fa4f7a4337)
and here add the using

Reply:
Okay, I'll get this sorted